ci/README.md

```diff
@@ -0,0 +1,40 @@
+# CLI commands for CI process
```

Comment:
I'm not sure we need a `README`, and I'm concerned about bitrot. People rarely remember to correct documentation which is in other files... This is not intended for consumption by end users, so I think these descriptions would fit better inside the respective files.

Reply:
As for me `README` is more user-friendly so you can easily read pre-formated version (headers examples etc) of the documentation. And From single responsibility prospective scripts are not responsible for usage docs but for implementations only and also it is easy to understand all implementation from single-source root.

Reply:
done